### DISCUSSION TITLE
Issue with Bag Borders

### DISCUSSION THREAD

**Discussion by MrNoodle1** [SFX-WoW/Masque#177] [Issues] [answered]
![bag1](https://user-images.githubusercontent.com/10471693/39316971-7c9ab5d0-4948-11e8-9908-e33351060b66.png)
This is what the border of my bags usually look like, but when I loot an item, it looks like this: 
![bag2](https://user-images.githubusercontent.com/10471693/39317034-9fef3736-4948-11e8-8800-520bdc786b3d.png)

Then I have to go to masque in my addons and disable bag borders and re-enable them. 

Does anyone have this issue or know how to fix it?

**Comment by StormFX** [comment, marked as answer]
Looks like the add-on you're using for your bag bar is either resizing some properties of the buttons or isn't telling Masque to update when an item is added.
